B827EBFFFE0678CD.json

```diff
@@ -0,0 +1,19 @@
+
+	{
++   " gateway_conf " : {
++     " gateway_ID " : " B827EBFFFE0678CD " ,
++     " servidores " : [
++ {
++         " server_address " : " ttn-router-us-west " ,
```

Comment:
That's not the correct URL for the US router, check the list here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses

Also make sure to remove all training spaces.